src/index.ts

```diff
@@ -194,4 +195,67 @@ function countTokens(encoding: Tiktoken, text: string | undefined): number {
   return encoding.encode(text).length;
 }
 
+async function countImageTokens(contentPart: OpenAI.Chat.ChatCompletionContentPartImage, chatModel: string): Promise<number> {
+  const visionRatio = (chatModel !== 'gpt-4o-mini') ? 1 : gpt4oMiniRatio;
```

Comment:
Consider using a Map to store chatModel-visionRatio pairs for better scalability and readability. Implement a fallback value of 1 when the chatModel is not found in the Map. Example:

```javascript
const visionRatioMap = new Map([
  ['gpt-4o-mini', gpt4oMiniRatio],
  // Add other models here
]);

const visionRatio = visionRatioMap.get(chatModel) ?? 1;
```

This approach allows for easier addition of new models and improves code maintainability.